roles/addons/nhc/templates/nhc.conf.j2

```diff
@@ -0,0 +1,39 @@
+#jinja2: lstrip_blocks: True
+#### Blue Banquise file ####
```

Comment:
is it Blue Banquise or BlueBanquise?

Reply:
BlueBanquise

Mistake here, due to auto correct.